desk/app/diary.hoon

```diff
@@ -144,29 +149,26 @@
   --
 ::
 ++  watch-epic
-  |=  her=ship
+  |=  [her=ship leave=?]
```

Comment:
If I understand this PR correctly: we may previously not have always subscribed to a publisher's epic/saga, but that is important, so now we make sure it's set up for everyone we subscribe to? Then, what is the `leave=&` case here for? Is it just so we can indiscriminately re-set-up these subscriptions (without getting "duplicate wire on sub" printouts) or do we actually care about getting the initial/current subscription result again?

Reply:
It's both. We need the initial data because we likely didn't have it and we want to tell people right away if they're mismatched, and I'm also trying to start cleaning up the dojo output.